pkg/crc/preflight/preflight_checks_tray_windows.go

```diff
@@ -19,126 +16,82 @@ import (
 )
 
 func checkIfTrayInstalled() error {
-	/* We want to force installation whenever setup is ran
-	 * as we want the service config to point to the executable
-	 * with which the setup command was issued.
-	 */
-
-	return fmt.Errorf("Ignoring check and forcing installation of System Tray")
+	// Check if tray exists in the startup folder
+	// And version of the tray is matching with expected version
+	return fmt.Errorf("Ignoring check, forcing installation of tray and daemon")
 }
 
-func fixTrayInstalled() error {
-	/* To avoid asking for elevated privileges again and again
-	 * this takes care of all the steps needed to have a running
-	 * tray by invoking a ps script which does the following:
-	 * a) Add logon as service permission for the user
-	 * b) Create the daemon service and start it
-	 * c) Add tray to startup folder and start it
-	 */
-
-	tempDir, err := ioutil.TempDir("", "crc")
+func fixDaemonInstalled() error {
+	currentExecutablePath, err := goos.Executable()
 	if err != nil {
-		logging.Error("Failed creating temporary directory for tray installation")
-		return err
-	}
-
-	defer func() {
-		_ = goos.RemoveAll(tempDir)
-	}()
-
-	// prepare the ps script
-	binPath, err := goos.Executable()
+		return fmt.Errorf("Failed to find current executables path: %w", err)
+	}
+
+	// Create the Posh script to launch daemon in the background with hidden window
```

Comment:
Regarding this hidden window thing, did you try https://stackoverflow.com/questions/23250505/how-do-i-create-an-executable-from-golang-that-doesnt-open-a-console-window-whe

Reply:
Tried this, but it doesn't seem to help with our situation..
- to run the daemon we need the command `crc daemon` so, we'll still need to have a posh script and a batch file to drop in the startup folder
- after compiling with `windowsgui` it doesn't show o/p in the console, so we can't use the same binary for CLI

Reply:
Ah the second point rules this approach out :)